documentation/modules/configuring/proc-resizing-persistent-volumes.adoc

```diff
@@ -6,8 +6,9 @@
 = Resizing persistent volumes
 
 [role="_abstract"]
-You can provision increased storage capacity by increasing the size of the persistent volumes used by an existing Strimzi cluster.
-Resizing persistent volumes is supported in clusters that use either a single persistent volume or multiple persistent volumes in a JBOD storage configuration.
+Persistent volumes used by a Strimzi cluster can be resized without any risk of data loss. 
+This capability allows for expanding storage capacity by increasing the size of existing persistent volumes. 
+Resizing is supported in clusters with either a single persistent volume or multiple persistent volumes configured in JBOD storage.
```

Comment:
```suggestion
Storage expansion is supported in clusters with either a single persistent volume per-broker or multiple persistent volumes per-broker configured with JBOD. Storage reduction is only supported with JBOD, after all partitions have been moved to other volumes of the same broker (intra-broker) or other brokers of the same cluster (intra-cluster).
```

Reply:
I would not mix removing JBOD volumes into this. It is something completely different than resizing volumes.

Reply:
> I would not mix removing JBOD volumes into this. It is something completely different than resizing volumes.

Reducing capacity is resizing. I think it is worth mentioning, also considering that the "Removing volumes from JBOD storage" rocedure is explained few paragraph below.

Reply:
Its not resizing of persistent volumes which this procedure is about.

Reply:
Ok, I see what you mean. Let's remove "Storage reduction..", but I would keep the rest.

Reply:
we do have a note after this intro that talks about decreasing the size of persistent volumes. I can expand that to give this content.

Reply:
I've replaced what we had in the note with this, as it was contradicting what we had originally.

Reply:
See what you think it the update. We can remove the note if it gets in the way.

Reply:
Sorry, but I think this is confusing as it suggests you can empty the disk and then shrink it. You should at least add there that the _reduction_ happens through _removing the whole disk_ and not by shrinking it.

Reply:
I've changed the note. We're talking about resizing in this procedure and showing how to expand, but I think a note on reduction is helpful.

> NOTE: Storage reduction is only possible with JBOD. You can remove a disk after moving all partitions on the disk to other volumes within the same broker (intra-broker) or to other brokers within the same cluster (intra-cluster).